packages/untp-test-suite/src/interfaces/utils/credentials.ts

```diff
@@ -8,10 +8,18 @@ export interface ICredentialFileData {
     type: string;
     version: string | null;
     dataPath: string;
-    url?: string | null
+    url?: string | null;
   }[];
 }
 
+// Current UNTP data models.
+const untpDefaultModel = new Set([
+  'traceabilityEvents',
+  'productPassport',
+  'digitalFacilityRecord',
+  'conformityCredential',
+]);
+
```

Comment:
Should we update these event names to reflect the latest changes in the schema folder structure? Since I noticed you have updated the schema folder names by adding a 'digital' prefix.